### PR TITLE
Header and Home Page

### PR DESCRIPTION
**Pull request by iamharnad**
Implemented Header page with search bar and Home page with sample products. 

- created a seperate component for Header
- created a seperate component for Home
- Dynamically calling product information using props

### CODE REVIEW

**Review by anjali1102 — APPROVED** [review]
few suggestions 
css variables 
className naming could be  shorter

**Review by aqsaaqeel — APPROVED** [review]
I like how you've passed the product attributes as props

File: src/Header.css [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  z-index: 100;
+}
+
+.header__logo {
+  display: flex;
+  align-items: center;
+  color: #ff9f00;
+  margin: 0 25px;
+}
+
```
Comment (Mahi24-prog) [inline src/Header.css:17]: Define flex properties once and use it where you want.

File: src/Header.css [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  padding: 10px;
+  border: none;
+  width: 100%;
+}
+
+.header__searchIcon {
+  padding: 5px;
+  height: 22px !important;
+  background-color: #ff9f00;
+}
```
Comment (Mahi24-prog) [inline src/Header.css:47]: use color variables

File: src/Header.css [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+  font-weight: 800;
+}
+
+.nav__itemBasket {
+  display: flex;
+  align-items: center;
+  color: white;
+  margin-right: 50px;
+  margin-left: 10px;
+}
```
Comment (Mahi24-prog) [inline src/Header.css:77]: you are writing flex properties repetitively. Define it once

**Review by Mahi24-prog — APPROVED** [review]
Go through suggestion once. Other than that looks good

**Pull request merged**